### PR TITLE
Bug 1198986 - (v1.0) Added about:rights to settings screen

### PR DESCRIPTION
**Pull request by sleroux**
Added aboutRights.xhtml from Android along with associated .dtd files so we can use the existing localization for those files if we need to.

### CODE REVIEW

File: Client/Assets/About/aboutRights.xhtml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        <div id="webservices-container">
+            <a name="webservices"/>
+            <h3>&rights2.webservices-header;</h3>
+
+            <p>&rights2.webservices-a;<a href="about:rights#disabling-webservices" onclick="showDisablingServices();">&rights2.webservices-b;</a>&rights3.webservices-c;</p>
+
+            <div id="disabling-webservices-container" style="margin-left:40px;">
+                <a name="disabling-webservices"/>
+                <!-- Safe Browsing cannot be disabled in Firefox Mobile; these instructions show how to do it on desktop
+                 <p><strong>&rights.safebrowsing-a;</strong>&rights.safebrowsing-b;</p>
```
Comment (rnewman) [inline Client/Assets/About/aboutRights.xhtml]: Make sure this file matches up with Bug 1196768.

File: Client/Assets/About/brand.dtd [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<!ENTITY  brandShorterName      "Firefox">
```
Comment (rnewman) [inline Client/Assets/About/brand.dtd]: We really ought to make these build parameters.

**Comment by rnewman** [comment]
LGTM. Don't forget the master version.

File: Client/Assets/About/brand.dtd (hunk 2 above)
Comment (sleroux) [inline Client/Assets/About/brand.dtd]: Can I file another bug for consolidating brand stuff?

**Comment by sleroux** [comment]
Added an additional patch that un-hides the more info section at the bottom to match the copy in https://bugzilla.mozilla.org/show_bug.cgi?id=1196768

File: Client/Assets/About/brand.dtd (hunk 2 above)
Comment (rnewman) [inline Client/Assets/About/brand.dtd]: Yes!

Comment (sleroux) [inline Client/Assets/About/brand.dtd]: https://bugzilla.mozilla.org/show_bug.cgi?id=1200386

**Pull request merged**